vendor/ember/shims.js

```diff
@@ -0,0 +1,247 @@
+        'default': Ember.inject.controller
+      },
+      'ember-controller/proxy': {
+        'default': Ember.ArrayProxy
+      },
+      'ember-controllers/sortable': {
+        'default': Ember.SortableMixin
+      },
+      'ember-debug': {
+        'log':      Ember.debug,
+        'inspect':  Ember.inspect,
+        'run':      Ember.runInDebug,
+        'warn':     Ember.warn
+      },
+      'ember-debug/container-debug-adapter': {
+        'default': Ember.ContainerDebugAdapter
+      },
+      'ember-debug/data-adapter': {
+        'default': Ember.DataAdapter
+      },
+      'ember-deprecations': {
+        'deprecate':      Ember.deprecate,
+        'deprecateFunc':  Ember.deprecateFunc
+      },
+      'ember-enumerable': {
+        'default': Ember.Enumerable
+      },
+      'ember-evented': {
+        'default': Ember.Evented
+      },
+      'ember-evented/on': {
+        'default': Ember.on
+      },
+      'ember-globals-resolver': {
+        'default': Ember.DefaultResolver
+      },
+      'ember-helper': {
+        'default':  Ember.Helper,
+        'helper':   Ember.Helper && Ember.Helper.helper
+      },
+      'ember-instrumentation': {
+        'instrument':   Ember.Instrumentation.instrument,
+        'reset':        Ember.Instrumentation.reset,
+        'subscribe':    Ember.Instrumentation.subscribe,
+        'unsubscribe':  Ember.Instrumentation.unsubscribe
+      },
+      'ember-locations/hash': {
+        'default': Ember.HashLocation
+      },
+      'ember-locations/history': {
+        'default': Ember.HistoryLocation
+      },
+      'ember-locations/none': {
+        'default': Ember.NoneLocation
+      },
+      'ember-map': {
+        'default':      Ember.Map,
+        'withDefault':  Ember.MapWithDefault
+      },
+      'ember-metal/destroy': {
+        'default': Ember.destroy
+      },
+      'ember-metal/events': {
+        'addListener':    Ember.addListener,
+        'removeListener': Ember.removeListener,
+        'send':           Ember.sendEvent
+      },
+      'ember-metal/get': {
+        'default': Ember.get
+      },
+      'ember-metal/mixin': {
+        'default': Ember.Mixin
+      },
+      'ember-metal/observer': {
+        'default':        Ember.observer,
+        'addObserver':    Ember.addObserver,
+        'removeObserver': Ember.removeObserver
+      },
+      'ember-metal/on-load': {
+        'default':  Ember.onLoad,
+        'run':      Ember.runLoadHooks
+      },
+      'ember-metal/set': {
+        'default':        Ember.set,
+        'setProperties':  Ember.setProperties,
+        'trySet':         Ember.trySet
+      },
+      'ember-metal/utils': {
+        'aliasMethod':  Ember.aliasMethod,
+        'assert':       Ember.assert,
+        'cacheFor':     Ember.cacheFor,
+        'copy':         Ember.copy
+      },
+      'ember-object': {
+        'default': Ember.Object
+      },
+      'ember-platform': {
+        'assign':         Ember.merge,
```

Comment:
This is also suspect. `Ember.merge` is deprecated and we now have `Ember.assign`, which should certainly be used here instead.